src/main/kotlin/br/unipar/plano/interfaces/rest/cobrancas/impl/CobrancaResourceImpl.kt

```diff
@@ -0,0 +1,65 @@
+package br.unipar.plano.interfaces.rest.cobrancas.impl
+
+import br.unipar.plano.domain.cobrancas.model.IdCobranca
+import br.unipar.plano.domain.cobrancas.service.CobrancaService
+import br.unipar.plano.interfaces.rest.cobrancas.CobrancaDetailsDTO
+import br.unipar.plano.interfaces.rest.cobrancas.CobrancaResource
+import br.unipar.plano.interfaces.rest.cobrancas.CobrancaSummaryDTO
+import br.unipar.plano.interfaces.rest.cobrancas.RegistrarCobrancaDTO
+import io.swagger.v3.oas.annotations.Operation
+import io.swagger.v3.oas.annotations.responses.ApiResponse
+import io.swagger.v3.oas.annotations.responses.ApiResponses
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.*
+import org.springframework.web.servlet.support.ServletUriComponentsBuilder
+import java.util.*
+import javax.validation.Valid
+
+@RestController
+@RequestMapping("/cobrancas")
```

Comment:
[Opcional] Como a cobrança é lançada para um contrato.

```suggestion
@RequestMapping("/contratos/{idContraro}/cobrancas")
```